gateway/1.8/modules/ROOT/pages/flex-architecture-high-availability.adoc

```diff
@@ -0,0 +1,94 @@
+= Flex Gateway Multiple Region Deployments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Install Flex Gateway in multiple regions to reduce latency and ensure live failover. You can configure most implementations with either multiple Flex Gateways or multiple Flex Replicas. Dependent on which you chose, there are different considerations for Redis Shared-Storage, policies, and logging. To learn more about Flex Replicas, see xref:index.adoc#replicas[Flex Replicas].
+
+All the following diagrams show Flex Gateway running in Connected Mode. For Flex Gateway running in Local Mode, the only difference is that Anypoint Platform would not be present in the deployment model. This does not affect how information flows between Flex Gateway and the upstream and downstream services.
+
+The implementation diagrams use Kubernetes terminology, but you can extend the architecture to other technology stacks that have the appropriate network controls applied.
+
+[[implementation-1]]
+== Implementation 1: High Availability with the Same APIs in Two Separate Regions
+
+Implementation 1 provides high availability for non-region specific traffic. The DNS service is configured for active-active failover meaning each region services traffic at all times.
+
+[[implementation-1a]]
+=== Implementation 1A: Cross-Regional Flex Gateway
+
+In this implementation:
+
+* A single Flex Gateway has Flex Replicas distributed across multiple region.
+* The Flex Gateway is registered once in Anypoint Platform and deployed across multiple regions and availability zones.
+* Logs and metrics for each Flex Gateway are consolidated in Anypoint Platform.
+* The DNS service is configured for active-active failover to direct traffic to any region at all times.
+* APIs deployed to the Gateway are available in all regions and availability zones. 
+* API configurations including upstream services and policies are the same for all regions. Upstream services must be region-agnostic meaning either the DNS is local or it lacks any reference to the region.
+* The Redis cluster should be replicated across regions or Redis shared storage can be used. Redis replication is required when traffic is not region specific.   
+* The only cross-regional traffic is Redis synchronization.
+
+In the following diagram, Flex Replicas in different regions and availability zones have distinct Kubernetes deployments:
+
+image:Flex-Gateway-DR-1.png["A detailed view of implementation 1A, which contains the necessary services to support High Availability in different regions Flex Replicas."]
+
+
+[[implementation-1b]]
+=== Implementation 1B: Region-Specific Flex Gateway
+
+In this implementation:
+
+* Each region has a distinct Flex Gateway with all resources as independent entities.
+* A separate Flex Gateway is registered for each region.
+* Logs and metrics are specific to their region because each Flex Gateway is a distinct entity in Anypoint Platform.
```

Comment:
I would add here: 
Each region specific API will have a distinct set of metrics, alerts, logs in the Anypoint Platform
Policies are also region specific even if they have a similar configuration because they apply to the API instance that is also region specific.